e2eAIOK/DeNas/cv/utils/cnn.py

```diff
@@ -44,8 +44,9 @@ def cnn_mutation_random_func(top_candidates, super_net, search_space, num_classe
     if len(top_candidates) <= 10:
         tmp_random_structure_str = plainnet_struct
     else:
-        tmp_idx = random.randint(0, len(top_candidates) - 1)
+        tmp_idx = random.choice(range(0, len(top_candidates) - 1))
```

Comment:
@csdingbin , does this fix means EA inconsistent issue is fixed?

Reply:
Yes. Change set() to dict.fromkeys(), the search space is same for different rounds